lib/controllers/AssetController.ts

```diff
@@ -46,17 +58,33 @@ export class AssetController extends CRUDController {
     const model = request.getBodyString('model');
     const reference = request.getBodyString('reference');
 
-    if (! request.input.resource._id) {
+    if (! request.input.args._id) {
```

Comment:
Why did you changed this? It allowed to create an asset with a specific ID in the "usual" way we are creating documents

Reply:
resource was marked as deprecated and told me to change it to intput.args

Reply:
Oh yes true, then you should rather use `request.getId()`

Reply:
I'll do that yes, always forgot theses helpers functions exists